apps/app/src/views/routes/Prescription.tsx

```diff
@@ -350,71 +290,90 @@ export const Prescription = () => {
                 <Text fontSize="md">{dispenseAsWritten ? 'Yes' : 'No'}</Text>
               )}
             </InfoGrid>
-
-            {orders.length === 0 ? null : (
+            <Divider />
+            <HStack justifyContent="space-between" width="100%">
+              <Text color="gray.500" fontWeight="medium" fontSize="sm">
+                Orders
+              </Text>
+              {loading ? null : (
+                <Button
+                  leftIcon={<FiPlus />}
+                  aria-label="New Order"
+                  as={RouterLink}
+                  to={`/orders/new?patientId=${patient?.id || ''}&prescriptionId=${id}`}
```

Comment:
possible to not add patientId param w/o id?

Reply:
Yup you can try it here [https://app.boson.health/orders/new](https://app.boson.health/orders/new) or [https://app.boson.health/orders/new?patientId=](https://app.boson.health/orders/new?patientId=)